src/server/pfs/drive/driver.go

```diff
@@ -488,6 +488,9 @@ func (d *driver) MakeDirectory(file *pfs.File, shard uint64) (retErr error) {
 		}
 		diffInfo.Appends[path.Clean(file.Path)] = _append
 	}
+	// The fact that this is a directory is signified by setting Children
+	// to non-nil
+	_append.Children = make(map[string]bool)
```

Comment:
Seems reasonable, was this something we should have been doing before or did the changes here require this?